src/PowerShellEditorServices.Protocol/Server/LanguageServer.cs

```diff
@@ -2010,8 +2010,13 @@ private static CompletionItemKind MapCompletionKind(CompletionType completionTyp
                 // This causes the editing cursor to be placed *before* the final quote after completion,
                 // which makes subsequent path completions work. See this part of the LSP spec for details:
                 // https://microsoft.github.io/language-server-protocol/specification#textDocument_completion
-                int len = completionDetails.CompletionText.Length;
-                completionText = completionDetails.CompletionText.Insert(len - 1, "$0");
+
+                // Since we want to use a "tab stop" we need to escape a few things for Textmate to render properly.
+                var sb = new StringBuilder(completionDetails.CompletionText)
+                    .Replace("\\", "\\\\")
```

Comment:
You might consider using a verbatim string here e.g. `.Replace(@"\", @"\\")` and below.

Reply:
done!

Reply:
Might as well use verbatim strings on lines 2017 & 2018 as well.  Makes the reading of "replaced this with that" easier if you don't have to mentally convert esc sequences.  :-)

Reply:
Yeah, yeah that's definitely nit-picking.  :-)  Overall LGTM.

Reply:
alright, they're everywhere now - and they're perfectly lined up!